### PR TITLE
CSV writer, refactored read functionality

### PR DESCRIPTION
**Pull request by SpookySquidward**
- Created a `csv_writer` object
- Created `review` object to store the title, body, and label of each review (sample) as strings
  - Refactored `csv_reader` and `csv_writer` to use new `review` data type
- `csv_reader`, `csv_writer`, and `review` are now all part of the `dataset` module
- Refactored `test_install.py` to demo the new read/write dynamics
- Fixed incorrect `scikit-learn` requirement per 6505b58
  - `sklearn` was being referenced instead, but that package name is deprecated in `pip`

### CODE REVIEW

**Comment by SpookySquidward** [comment]
Added error handling when a blank line is found in a `.csv` file opened with `csv_reader`; @frejaz is seeing `csv_writer` add newlines between each sample written, I haven't been able to replicate this behavior on my machine though

**Review by frejaz — APPROVED** [review]
✨
